Dockerfile.lmes-job

```diff
@@ -22,7 +22,6 @@ RUN python -c 'from lm_eval.tasks.unitxt import task; import os.path; print("cla
 
 ENV PYTHONPATH=/opt/app-root/src/.local/lib/python3.11/site-packages:/opt/app-root/src/lm-evaluation-harness:/opt/app-root/src:/opt/app-root/src/server
 ENV HF_HOME=/opt/app-root/src/hf_home
-ENV UNITXT_ARTIFACTORIES=/opt/app-root/src/my_catalogs
+ENV UNITXT_CATALOGS=/opt/app-root/src/my_catalogs
```

Comment:
Is this variable only used here in the Dockerfile or user can override it in some way?
Because if it "user facing" we might want to keep backward compatibility with a warning for a release or similar

Reply:
@danielezonca this is purely internal (i.e. specifying LMEval where to find additional catalogs), we don't expect the user to have change it or use it.